Code/Tools/ProjectManager/Source/ProjectButtonWidget.cpp

```diff
@@ -267,47 +325,151 @@ namespace O3DE::ProjectManager
         return m_projectInfo;
     }
 
-    void ProjectButton::RestoreDefaultState()
+    void ProjectButton::ShowLogs()
     {
-        m_projectImageLabel->SetEnabled(true);
-        m_projectImageLabel->SetOverlayText("");
-        m_projectMenuButton->setVisible(true);
+        QDesktopServices::openUrl(m_logUrl);
```

Comment:
In a future PR this is where we can PALify the action so on Windows we open CMTrace instead of Notepad.